build.sh

```diff
@@ -1134,8 +1139,11 @@ EOM
     "package-ios-swift")
         cd tightdb_objc
         for version in 8.0 8.1 8.2; do
-            REALM_XCODE_VERSION="$version" sh build.sh prelaunch-simulator
-            REALM_XCODE_VERSION="$version" sh build.sh ios-swift
+            REALM_XCODE_VERSION=$version
+            REALM_SWIFT_VERSION=
+            set_xcode_and_swift_versions
```

Comment:
No, doing that would fail with a complaint that later versions of Xcode do not support the selected (one for Xcode 8.0) version of Swift.